aztk/config.py

```diff
@@ -115,12 +94,16 @@ def _merge_dict(self, secrets_config):
             self.docker_password = docker_config.get('password')
 
         default_config = secrets_config.get('default')
-
         # Check for ssh keys if they are provided
         if default_config:
             self.ssh_priv_key = default_config.get('ssh_priv_key')
             self.ssh_pub_key = default_config.get('ssh_pub_key')
 
+    def load(self):
+        # read global ~/secrets.yaml
+        self._load_secrets_config(os.path.join(constants.HOME_DIRECTORY_PATH, '.aztk', 'ssh.yaml'))
```

Comment:
is this required? would it fail if there is no .aztk in my home dir?

Reply:
No, it won't fail. If the path provided does not exist, _load_secrets_config just returns. 

That is done here: https://github.com/jafreck/aztk/blob/feature/global-config/aztk/config.py#L64